pkgs/top-level/all-packages.nix

```diff
@@ -9296,7 +9296,7 @@ with pkgs;
     stdenv = gcc8Stdenv;
   };
 
-  hylafaxplus = callPackage ../servers/hylafaxplus { };
+  hylafaxplus = callPackage ../servers/hylafaxplus { libtiff = libtiff_4_5; };
```

Comment:
I suggest adding inline comment here why you need pinned version: just mention needed utilities here.

Otherwise looks great!

Reply:
Done.